cmd/smith/app/app.go

```diff
@@ -50,6 +70,7 @@ type App struct {
 	Plugins              []plugin.NewFunc
 	Workers              int
 	DisablePodPreset     bool
+	LeaderElectionConfig
```

Comment:
why not nest? Makes it easier to understand that a.RetryPeriod etc. is referring to leader election?

Reply:
Makes sense.